repository/g.json

```
@@ -1920,6 +1920,19 @@
 				}
 			]
 		},
+		{
+			"name": "GotoRubyGem",
+			"details": "https://github.com/ZainIftikhar7vals/GotoRubyGem",
+			"readme": "https://raw.githubusercontent.com/ZainIftikhar7vals/GotoRubyGem/main/README.md",
+			"labels": ["code navigation", "ruby gems"],
```

```suggestion
			"labels": ["code navigation", "ruby gems", "ruby"],
```
A general "ruby" label will be more useful actually.